### DISCUSSION TITLE
New Feature: Connectivity v2 feedback

### DISCUSSION THREAD

**Discussion by vincentmaxwell** [community/community#112351] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hello, I was using codespaces to teach my students at my school however since v2 launched in February, the feature prevented us from launching our codespaces. Our IT department whitelisted all domains and redirects, but I am still experiencing issues. At first the codespace would give an error screen saying the device was offline. Then after IT checked the webfilter and allowed redirects it is loading slowly and the console output is several errors regarding the relay. The error message it displays is:
`Failed to connect to relay url Code 1006 Reason: connection rejected by browser`

I have ensured that redirects on the browser are allowed for the site permissions and have tested different browsers. The IT department also assures me the webfilter is not causing this issue so I am at a loss?

The workaround currently is to disable the v2 connectivity feature.

**Comment by osortega** [comment]
Hello @vincentmaxwell 👋 !

I'm sorry for the friction caused here. In order to debug this further, do you see anything in the Network tab of the Developer tools in the browser? Or any errors in the console?

There should be some sort of indication of how the requests are being denied. 

Also, are you able to connect on any other network such as your wifi connection at home?

**Reply by vincentmaxwell** [reply]
![Untitled](https://github.com/community/community/assets/150384839/ab9cb2e3-2a49-45dc-ba77-9f1d2408adb3)

This is with the V2 Feature enabled. Does not happen when I disable that feature preview. I have not tried the V2 Feature at home but I would guess it works? Is there something else the IT department has to do on the browser specifically instead of the web filter? I made sure to allow redirects on the browser.

**Comment by MatuaDoc** [comment]
Not to "me too" but ... me too. I have to tell my students to disable this, as allow-listing the URL in the documentation doesn't do anything different.

**Comment by dreamorosi** [comment]
I have been experiencing issues and being unable to connect both via multiple browser and VSCode app when Connectivity v2 was enabled. 

I have now disabled it so that I can continue working, however if you need me to provide any diagnostics please reach out. Not sure what to report here.

**Comment by MatuaDoc** [comment]
The feature preview cannot be disabled anymore, as GitHub has removed the option before even this thread was resolved.

**Reply by dreamorosi** [reply]
I also don't see it anymore, and as a result I can't use CodeSpaces anymore

**Reply by MatuaDoc** [reply]
It's fairly disastrous for me as my students were right in the middle of an assignment. Now, they can only work on their assignment at home, wasting school hours. We don't have a dedicated IT department at my school, it's a team of one who has other issues to attend to and hasn't been able to solve the connectivity problem so far.

Our lifeline has gone.
